workers/unity/Assets/Gdk/Core/Systems/SpatialOSSendSystem.cs

```diff
@@ -19,13 +19,34 @@ internal static class Warnings
         private readonly List<int> registeredReplicators = new List<int>();
         private readonly List<int> commandSenders = new List<int>();
 
+        private int ticks = 0;
+        private const int METRIC_TICKS_UPDATE = 200;
+
+        public delegate Worker.Metrics MetricsComputation();
+        private MetricsComputation metricComputedEverySend;
+
         protected override void OnCreateManager(int capacity)
+        {
+            OnCreateManager(capacity, delegate()
+            {
+                float fps = 1.0f / Time.deltaTime;
+                return new Worker.Metrics
+                {
+                    // Load is 0 is fps is at most 20, 1 if it is at least 30, and linear in between.
+                    Load = Mathf.Min(1.0f, Mathf.Max(0.0f, 3.0f - 0.1f * fps))
```

Comment:
We shouldn't cap the load at 1. A load of 15 and a load of 1 means very different things :)

Reply:
Oh I just realised the load is inverted here. 0 load = high fps, 1 load = low fps :)

Reply:
This is a lot of magic numbers, would be nice to extract into constants.

Reply:
Done!